source/api/cypress-api/custom-commands.md

```diff
@@ -268,6 +268,28 @@ This is usually unnecessary because Cypress is already configured to swap out ba
 For more complex use cases feel free to overwrite existing commands.
 {% endnote %}
 
+### Overwrite `screenshot` command
+
+This example overwrites `screenshot` to always wait until a certain element is visible.
+
+```javascript
+Cypress.Commands.overwrite('screenshot', (originalFn, subject, name, options) => {
+
+  // call another command, no need to return as it is managed
+  cy.get('.app')
+    .should('be.visible')
+
+    // overwrite the default timeout, because screenshot does that internally
+    // otherwise the `then` is limited to the default command timeout
+    .then({ timeout: 45000 },
```

Comment:
I believe this should correspond to the `responseTimeout` defined in the user's configuration. You could access this as `Cypress.config('responseTimeout')`